### PR TITLE
Feature/QR Codes Admin

### PR DESCRIPTION
**Pull request by quan612**
+ Allow admin to see list of QR Codes
+ Add a new QR Code
+ Edit existing QR Code
+ Go to QR Claim page for end user
+ Go to QR Image page for end user

### CODE REVIEW

**Comment by quan612** [comment]
The first two commits are for backend.

The last commit is for React part

File: src/components/admin/reward/qr-codes/List/ResultTable.tsx [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+                        boxSize={{ base: '16px', xl: '24px' }}
+                        onClick={async () =>
+                          await pauseAction({
+                            id,
+                            isEnabled: isEnabled ? false : true,
+                          })
+                        }
+                        color="#89A4C2"
+                        _hover={{ cursor: 'pointer', color: '#00BBC7' }}
+                      >
```
Comment (ShahovaJane) [inline src/components/admin/reward/qr-codes/List/ResultTable.tsx:119]: can you use theme object for colors

**Review by ShahovaJane — CHANGES_REQUESTED** [review]

File: src/components/shared/Icons/index.js [hunk 2]
```diff
@@ -62,7 +75,7 @@ export const TwitterSettingIcon = () => {
   )
 }
 
-export const RewardsSettingIcon = () => {
+export const RewardsSettingIcon = ({ color = "#00BBC7" }) => {
```
[the remark this answers is not in the document]
Comment (quan612) [reply src/components/shared/Icons/index.js:78]: this one is a bit tricky, I will fix this later.

**Comment by quan612** [comment]
There are two request changes are not done yet, I will revisit later. We just need to build on staging to test today

File: src/components/admin/reward/landing/useGetRewardItems.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
+// UI
+import { useTheme } from '@chakra-ui/react'
+import {
```
Comment (ShahovaJane) [inline src/components/admin/reward/landing/useGetRewardItems.tsx:3]: // Components

**Review by ShahovaJane — CHANGES_REQUESTED** [review]

**Pull request merged**